src/Forms/Shared/Samples/Security/OAuth/OAuth.xaml.cs

```diff
@@ -264,4 +264,9 @@ public async Task<Credential> CreateCredentialAsync(CredentialRequestInfo info)
         }
 #endregion 
     }
+}urn completion source task so the caller can await completion.
```

Comment:
I think this kind of error is probably an issue with the python script. I think the sample generator script has logic to avoid this specific issue